.husky/pre-commit

```diff
@@ -0,0 +1 @@
+yarn test:run
```

Comment:
this will make commits very slow. please leave it up to the developer when to run tests locally.